src/pytorch_lightning/core/module.py

```diff
@@ -1479,6 +1479,34 @@ def clip_gradients(
             gradient_clip_val: The value at which to clip gradients.
             gradient_clip_algorithm: The gradient clipping algorithm to use. Pass ``gradient_clip_algorithm="value"``
                 to clip by value, and ``gradient_clip_algorithm="norm"`` to clip by norm.
+
+        Note:
+            For manual optimization (``self.automatic_optimization = False``), if you want to use gradient clipping,
+            consider calling ``self.clip_gradients(opt, gradient_clip_val, gradient_clip_algorithm)`` as shown in the
+            example (with a single optimizer) below,
+
+        Example::
+            from pytorch_lightning import LightningModule
```

Comment:
This example is great, but it goes into the user guide docs, which is what you have already done. So let's remove it from here, or shorten it to just the `training_step` method.

Reply:
Makes sense.